### PR TITLE
Fix metric for substrate_proposer_block_constructed_count

### PR DESCRIPTION
**Pull request by echevrier**
According to @haerdib suggestion:  
Init only Aura consensus at start up, relay chain consensus is not used

### CODE REVIEW

**Comment by brenzi** [comment]
is this tested with polkadot-launch?

**Comment by echevrier** [comment]
> is this tested with polkadot-launch?

Yes, actually only with polkadot-launch.

File: polkadot-parachains/src/service.rs [hunk 1]
```diff
@@ -649,112 +649,59 @@ where
 			let telemetry2 = telemetry.clone();
```
Comment (haerdib) [inline polkadot-parachains/src/service.rs:649]: If we remove the second initialization, I doubt we need to have all these `clones()`. Can you remove them?

**Review by mosonyi — APPROVED** [review]
All good!
Please bump the version numbers to be able to release it immediately.

File: polkadot-parachains/src/service.rs (hunk 1 above)
Comment (haerdib) [reply polkadot-parachains/src/service.rs:681]: Removed them, so we can proceed

**Review by brenzi — APPROVED** [review]

**Pull request merged**